src/index.ts

```diff
@@ -13,8 +13,8 @@ export function sortByDesc<T = any>(values: any, iteratees?: any): T[] {
     return sort(values).desc(iteratees);
 }
 
-export function orderBy<T = any>(values: T[], iteratees: string | string[], orders: string | string[]): T[] {
-    if (typeof iteratees === "string") {
+export function orderBy<T = any>(values: T[], iteratees: Function | string | any[], orders: string | string[]): T[] {
```

Comment:
```suggestion
export function orderBy<T = any>(values: T[], iteratees: Function | string | string[], orders: string | string[]): T[] {
```

Reply:
this wouldn't allow passing in multiple functions, see the last commit.